pkgs/servers/cloud9/default.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv ? (import <nixpkgs> {}).stdenv
+, lib ? (import <nixpkgs> {}).pkgs.lib
+, pkgs ? (import <nixpkgs> {}).pkgs
+, nodePackages ? (import <nixpkgs> {}).pkgs.nodePackages
+}:
+let
+inherit nodePackages;
+inherit lib;
+inherit pkgs;
```

Comment:
If you need these defaults for testing, you're doing it wrong, in my opinion. You ought to access the package via the top-level package attrset for a more realistic experience. But whatever floats your boat ...